packages/manager/src/features/Kubernetes/CreateCluster/HAControlPlane.tsx

```diff
@@ -0,0 +1,85 @@
+import { FormLabel } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import * as React from 'react';
+
+import { Box } from 'src/components/Box';
+import { displayPrice } from 'src/components/DisplayPrice';
+import { FormControl } from 'src/components/FormControl';
+import { FormControlLabel } from 'src/components/FormControlLabel';
+import { Link } from 'src/components/Link';
+import { Radio } from 'src/components/Radio/Radio';
+import { RadioGroup } from 'src/components/RadioGroup';
+import { Typography } from 'src/components/Typography';
+import { HIGH_AVAILABILITY_PRICE } from 'src/constants';
+
+export const HACopy = () => (
+  <Typography>
+    Recommended for prodcution workloads, a high availability (HA) control plane
+    is replicated on multiple master nodes to 99.99% uptime.
+    <br />
+    <Link to="https://www.linode.com/docs/guides/enable-lke-high-availability/">
+      Learn more about the HA control plane
+    </Link>
+    .
+  </Typography>
+);
+
+export interface Props {
+  highAvailability: boolean | undefined;
+  setHighAvailability: (ha: boolean | undefined) => void;
+}
+
+export const HAControlPlane = (props: Props) => {
+  const { setHighAvailability } = props;
+  const theme = useTheme();
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.value === 'yes') {
+      setHighAvailability(true);
+    } else {
+      setHighAvailability(false);
+    }
+  };
+
+  if (HIGH_AVAILABILITY_PRICE === undefined) {
+    return null;
+  }
+
+  return (
+    <Box>
+      <FormControl data-testid="ha-control-plane">
+        <FormLabel
+          sx={{
+            '&&.MuiFormLabel-root.Mui-focused': {
+              color: theme.name === 'dark' ? 'white' : theme.color.black,
+            },
+          }}
+          id="ha-radio-buttons-group-label"
```

Comment:
Optional, but instead of using the `useTheme` hook, you can pass a function to `sx`, which exposes the theme.

```suggestion
          sx={(theme) => ({
            '&&.MuiFormLabel-root.Mui-focused': {
              color: theme.name === 'dark' ? 'white' : theme.color.black,
            },
          })}
          id="ha-radio-buttons-group-label"
```

Reply:
I went ahead and removed the `useTheme` hook. Less things to import.